pkg/pillar/scripts/device-steps.sh

```diff
@@ -101,17 +107,74 @@ get_ntp_servers() {
     echo "$out"
 }
 
-# Return one (the first) ntp server with default if none
+# Return one (the first) ntp server or default if none.
```

Comment:
Can we pass all NTP servers to chrony or would it require too many changes in the code?

Reply:
(would be great to avoid the limitation on using only one NTP server that we have today)

Reply:
I would think that 1+2 in one list (with equal priority) would be the way to go (and 3 if 1+2 is empty).